modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -49,19 +48,21 @@ If you have external client providers such as PingFederate configured for your e
 
 *Best Practice*
 
-To secure your APIs by leveraging the multiple client providers feature, create one external client provider per environment. Assign the existing client provider to your production environment and add the new providers to other environments, such as sandbox, and QA. 
+To secure your APIs, create one external client provider per environment. Assign the existing client provider to your production environment and add the new providers to other environments, such as sandbox, and QA. However, check the possible impact on managed APIs and their consumers. 
 
-However, before changing any existing client providers, ensure that you check the impact on managed APIs and their consumers. Before you implement multiple client providers, see the <<guidelines,guidelines>>.
+Avoid using the same IdP in production and non-production environments. You can use the same IdP in multiple production environments, or in multiple non-production environments.
 
+Before implementing multiple client providers, see the <<guidelines,guidelines>>.
 
 [[guidelines]]
 == Guidelines for Implementing Multiple Client Providers
```

Comment:
Per previous comments, I suggest that we look further into the order and titling of subtopics here.
I would call these requirements: indeed, prerequisites to implementing multiple providers and something that should precede any instructions or be mentioned at the beginning of any conceptual descriptions.

Reply:
Thanks, Kim. Good point! I'm thinking that the Guidelines section doesn't really list prerequisites, but rather it lists best practices.